python/agents/data-science/tests/test_agents.py

```diff
@@ -31,24 +31,23 @@
 from data_science.sub_agents.bigquery.agent import database_agent
 
 session_service = InMemorySessionService()
-artifact_service = InMemoryArtifactService()
-
-
-class TestAgents(unittest.TestCase):
+artifact_service = InMemoryArtifactService() 
+class TestAgents(unittest.IsolatedAsyncioTestCase): 
```

Comment:
nit: leave two lines

```suggestion
artifact_service = InMemoryArtifactService()

  
class TestAgents(unittest.IsolatedAsyncioTestCase): 
```